packages/optimizers/image/Cargo.toml

```diff
@@ -2,16 +2,17 @@
 authors = ["Devon Govett <devongovett@gmail.com>"]
 name = "parcel-image"
 version = "0.1.0"
+edition = "2021"
 
 [lib]
 crate-type = ["cdylib"]
 
 [dependencies]
-napi = {version = "2.3.3", default-features = false, features = ["napi4", "compat-mode"]}
-napi-derive = "2"
+napi = {version = "2.10.6", features = ["napi4", "compat-mode"]}
```

Comment:
```suggestion
napi = {version = "2.10.6", features = ["napi4"]}
```

Reply:
Not sure it makes a difference but I made it match.